.devcontainer/devcontainer.json

```diff
@@ -5,12 +5,19 @@
     // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
     "image": "mcr.microsoft.com/devcontainers/base:bookworm",
     "features": {
-        "ghcr.io/devcontainers/features/docker-inside-docker:1": {},
-        "ghcr.io/devcontainers/features/git:1": {},
+        "ghcr.io/devcontainers/features/docker-in-docker:2": {},
         "ghcr.io/devcontainers/features/kubectl-helm-minikube:1": {},
         "ghcr.io/devcontainers/features/python:1": {"version": "3.11"}
-    }
-
+    },
+    "customizations": {
+        "vscode": {
+            "settings": {
+                "python.defaultInterpreterPath": ".venv/bin/python"
+            }
+        }
+    },
+    "postCreateCommand": "make poetry_setup; make dev_setup",
```

Comment:
Simplify: `make poetry_setup dev_setup`

Reply:
Simplification cannot take place since make cannot identify the two commands (check the output below).

```
$ make poetry_setup make dev_setup
curl -sSL https://install.python-poetry.org | python - --version 1.7.1
Retrieving Poetry metadata

The latest version (1.7.1) is already installed.
poetry config virtualenvs.in-project true --local
make: *** No rule to make target 'make'.  Stop.
```